synapse/lib/stormlib/gen.py

```diff
@@ -66,23 +97,221 @@ class LibGen(s_stormtypes.Lib):
             [ ou:industry=* :name=$name ]
             return($node)
         }
+
         function newsByUrl(url) {
             media:news:url=$url
             return($node)
             [ media:news=* :url=$url ]
             return($node)
         }
+
         function softByName(name) {
             it:prod:softname=$name
             -> it:prod:soft
             return($node)
             [ it:prod:soft=* :name=$name ]
             return($node)
         }
+
         function vulnByCve(cve) {
             risk:vuln:cve=$cve
             return($node)
             [ risk:vuln=* :cve=$cve ]
             return($node)
         }
+
+        function riskThreat(name, reporter) {
+            ou:name=$name
+            tee { -> risk:threat:org:name } { -> risk:threat:org:names } |
+            +:reporter:name=$reporter
+            return($node)
+
+            [ risk:threat=*
+                :org:name=$name
+                :reporter = { yield $orgByName($name) }
```

Comment:
looks like this should be `:org` instead of `:reporter`

Reply:
Thanks! It should have been `orgByName($reporter)` great catch!